src/modules/events/models/Event.js

```diff
@@ -0,0 +1,16 @@
+module.exports = ({
+  knex = require('../../../config/database'),
+  tableName = 'evento'
+}) => {
```

Comment:
```suggestion
const knex = require('../../../config/database');

module.exports = () => {
```
Não imagino outro uso para receber o `knex` como parâmetro, tem algum?

E o nome da tabela pode ser uma constante no código mesmo, acredito que seja um mal uso conseguir alterar o nome da tabela sendo que o modelo é próprio para eventos.

Reply:
A forma como o Knex consegue passar esses parâmetros para a função da query é essa. Vou testar dessa outra forma, mas é possível que não funcione.

Oks. Tinha feito dessa forma pois ainda achava que seria um modelo reutilizado para as outras models.

Reply:
Fiz a mudança e deu certo.